### DISCUSSION TITLE
newbie... trying to add meta keywords but not showing up, am i doing it wrong?

### DISCUSSION THREAD

**Discussion by brianloweongit** [11ty/eleventy#2970] [Q&A] [answered]
For a specific page, in front matter I have 
```
---
title: My unique title
keywords: lorem, ipsum, dolor, samet
---
```
My page template calls base.html which has 
```
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>{{ title }}</title>
    <meta name="keywords" content="{{ keywords }}" />
```
... etc.
The HTML page generated has a <head> with a <title> having My unique title, but there's no meta tag named keywords.

Can anyone tell me what I'm doing wrong?

**Comment by pdehaan** [comment, marked as answer]
I'm unable to reproduce w/ Eleventy 2.0.1:

### src/index.liquid
```liquid
---
title: My unique title
keywords: lorem, ipsum, dolor, samet
layout: base.html
---

<p>THis is some content</p>
```

### src/_includes/base.html
```liquid
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>{{ title }}</title>
    <meta name="keywords" content="{{ keywords }}" />
  </head>
  <body>

    <main>{{ content }}</main>

  </body>
</html>
```


### OUTPUT

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>My unique title</title>
    <meta name="keywords" content="lorem, ipsum, dolor, samet" />
  </head>
  <body>
    <main>
      <p>THis is some content</p>
    </main>
  </body>
</html>
```

```sh
npm run build 


> 11ty-2970@1.0.0 build
> eleventy

[11ty] Writing www/index.html from ./src/index.liquid
[11ty] Wrote 1 file in 0.02 seconds (v2.0.1)
```

**Reply by brianloweongit** [reply]
Thanks for trying. I've now found another version of base.html that simply wasn't adding the meta tag, and buried in some config file was an instruction for my page to use the old version of base.html - so it's just a stupid newbie mistake of not properly using the config cascades, and keeping old versions of files around /:
Thanks for your time.
